### ISSUE TITLE
How is the training set and the test set of the experiment divided?

### ISSUE THREAD

**Issue by xuliuwei** [xiehousheng/SemLA#3]

**Comment by xiehousheng** [comment]
Thank you for your interest in our work. In our method, the training set uses the COCO dataset, which contains approximately 240000 pairs of images. For the test set, we have chosen some existing well aligned public datasets, such as MSRS, M3FD. You can find more details in our paper, such as the implementation details of the test set and the download address of the dataset.

**Comment by xuliuwei** [comment]
Thank you for your reply. Would you also like to ask which year your COCO data set is specifically used? Are you comfortable publishing your test set and related transformations to the test set?

**Comment by xiehousheng** [comment]
We selected the latest COCO training set and all images from the test set as the training set for SemLA. In addition, we have published the code for generating the test set composite deformation in SemLA\assets\data_transform.py and our test set is generated based on this code and public dataset.

**Comment by xuliuwei** [comment]
Can you provide the test set you used? I'd like to replicate the results of your paper

**Comment by xuliuwei** [comment]
Can you provide the test set you used? I'd like to replicate the results of your paper.

**Comment by xiehousheng** [comment]
We do not intend to provide this part of the data as it can be easily generated from the code and dataset links we provide, making it easier to verify the performance of SemLA.
